src/Neo/Extensions/SmartContract/ContractStateExtensions.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// ContractStateExtensions.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.Extensions.SmartContract
+{
+    public static class ContractStateExtensions
+    {
+        /// <summary>
+        /// Get Storage value by storage map key.
+        /// </summary>
+        /// <param name="contractState"></param>
+        /// <param name="snapshot">Snapshot of the database.</param>
+        /// <param name="storageKey">Key in the storage map.</param>
+        /// <returns>Storage value of the item.</returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static StorageItem GetStorage(this ContractState contractState, DataCache snapshot, byte[] storageKey)
```

Comment:
May be rename to `StorageGet` (an analogue of `System.Storage.Get`)?

Reply:
I would like to keep the `naming a convention` of `dotnet`